lib/requester/browser/request.js

```diff
@@ -128,7 +128,11 @@ function request(options, callback) {
         }
         if((new RegExp(METHODS_WITH_BODY.join('|'), 'i')).test(options.method)){
             var encoding = (options.encoding || 'application/x-www-form-urlencoded').toLowerCase();
-            options.headers['content-type'] = encoding;
+
+            if (!options.headers['content-type'] && !options.headers['Content-Type']) {
+                options.headers['content-type'] = encoding;
+            }
+
```

Comment:
Looks Good. Do note: This fix also needs to be present on `Postman-request` to ensure consistency.